### ISSUE TITLE
Cannot control fan settings on Gigabyte 7900GRE

### ISSUE THREAD

**Issue by m-czernek** [ilya-zlobintsev/LACT#344]
### Checklist

- [X] The problem is not listed in the [hardware support matrix](https://github.com/ilya-zlobintsev/LACT?tab=readme-ov-file#hardware-support) as a known limitation
- [X] I've included a [debug snapshot](https://github.com/ilya-zlobintsev/LACT?tab=readme-ov-file#reporting-issues) if the issue is related to hardware configuration

### Bug description

When using LACT, I cannot control fans on my AMD Radeon RX7900 GRE. See screenshot, I set the fans to 100% and they are still 0%.

Attaching a debug snapshot at: [LACT-sysfs-snapshot-20240624-195418.tar.gz](https://github.com/user-attachments/files/15959554/LACT-sysfs-snapshot-20240624-195418.tar.gz)

![image](https://github.com/ilya-zlobintsev/LACT/assets/16302538/88a31ba4-caa1-48eb-be85-d66bb812a9a2)

### System info

```markdown
- LACT version: 0.5.4-release (commit 6dc3520)
- GPU model: Radeon RX 7900 XT/7900 XTX/7900M (https://www.gigabyte.com/Graphics-Card/GV-R79GREGAMING-OC-16GD#kf)
- Kernel version: 6.9.5-1-default
- Distribution: openSUSE Tumbleweed
```

**Comment by ilya-zlobintsev** [comment]
Could you try the [test release](https://github.com/ilya-zlobintsev/LACT/releases/tag/test-build) and see if it fixes the issue? This might be related to #329 .

**Comment by m-czernek** [comment]
In the meantime, I'm trying a different GPU. I reproduced this on RX 7700 as well, the test release didn't help. I have:

- Updated the RPM (`lact-0.5.5-0.x86_64`)
- Restarted the `lactd` service

Same behavior with fans.

I also see a very strange behavior that the GPU, when I plug it into the system, is power throttled. The max power usage limit doesn't change between the 7900GRE and 7700, it tops at `253W`. When I set it to 253W, my GPU takes at most 91W, and seems to underperform.

I wonder, could this be something else?

**Comment by m-czernek** [comment]
Also note btw, regarding the fan issue itself, the static settings does kick in, but only when under load. My assumption is that the GPU waits until certain temperature, and then activates the fan settings.

**Comment by ilya-zlobintsev** [comment]
>  the static settings does kick in, but only when under load. 

This is known behaviour. From readme:
> There is an unconfigurable temperature threshold below which the fan does not get turned on, even with a custom curve.

This affects static speed too as it's applied as a curve with a single point.

**Comment by m-czernek** [comment]
Gotcha, sorry for the noise then. Before closing the issue, have you encountered the power issue by any chance?

**Comment by ilya-zlobintsev** [comment]
There have been reports of the power limit being lower than it should be: https://gitlab.freedesktop.org/drm/amd/-/issues/2356. However, it was supposed to be fixed in kernel 6.8 (see #255 for more info), so I'm not sure why you're still getting this issue.

The drm/amd tracker might have more reports or other useful info about this.

**Comment by m-czernek** [comment]
Thank you very much for your help, appreciate the links. I'm closing the issue since the fan setting is WAD.
